packages/@react-spectrum/label/stories/StaticField.stories.tsx

```diff
@@ -0,0 +1,125 @@
+
+export let Default: StaticFieldStory = {
+  args: {label: 'Test', value: 'This is some sample text'},
+  name: 'String'
+};
+
+export let StringArray: StaticFieldStory = {
+  ...Default,
+  args: {...Default.args, value: ['wow', 'cool', 'awesome']},
+  name: 'String array'
+};
+
+export let CalendarDateType: StaticFieldStory = {
+  ...Default,
+  args: {...Default.args, value: new CalendarDate(2019, 6, 5), formatOptions: {dateStyle: 'medium'}},
+  name: 'CalendarDate'
+};
+
+export let CalendarDateTimeType: StaticFieldStory = {
+  ...Default,
+  args: {...Default.args, value: new CalendarDateTime(2020, 2, 3, 12, 23, 24, 120), formatOptions: {dateStyle: 'medium', timeStyle: 'medium'}},
+  name: 'CalendarDateTime'
+};
+
+export let ZonedDateTimeType: StaticFieldStory = {
+  ...Default,
+  args: {...Default.args, value: new ZonedDateTime(2020, 2, 3, 'America/Los_Angeles', -28800000), formatOptions: {dateStyle: 'long', timeStyle: 'long'}},
+  name: 'ZonedDateTime'
+};
+
+export let DateType: StaticFieldStory = {
+  ...Default,
+  args: {...Default.args, value: new Date(2000, 5, 5), formatOptions: {dateStyle: 'long'}},
+  name: 'Date'
+};
+
+export let TimeType: StaticFieldStory = {
+  ...Default,
+  args: {...Default.args, value: new Time(9, 45), formatOptions: {timeStyle: 'short'}},
+  name: 'Time'
+};
+
+export let CalendarDateRange: StaticFieldStory = {
+  ...Default,
+  args: {...Default.args, value: {start: new CalendarDate(2019, 6, 5), end: new CalendarDate(2019, 7, 5)}, formatOptions: {dateStyle: 'medium'}},
+  name: 'RangeValue<CalendarDate>'
+};
+
+export let CalendarDateTimeRange: StaticFieldStory = {
+  ...Default,
+  args: {...Default.args, value: {start: new CalendarDateTime(2020, 2, 3, 12, 23, 24, 120), end: new CalendarDateTime(2020, 3, 3, 12, 23, 24, 120)}, formatOptions: {dateStyle: 'medium', timeStyle: 'medium'}},
+  name: 'RangeValue<CalendarDateTime>'
+};
+
+export let ZonedDateTimeRange: StaticFieldStory = {
+  ...Default,
+  args: {...Default.args, value: {start: new ZonedDateTime(2020, 2, 3, 'America/Los_Angeles', -28800000), end: new ZonedDateTime(2020, 3, 3, 'America/Los_Angeles', -28800000)}, formatOptions: {dateStyle: 'medium', timeStyle: 'medium'}},
+  name: 'RangeValue<ZonedDateTime>'
+};
+
+export let DateRange: StaticFieldStory = {
+  ...Default,
+  args: {...Default.args, value: {start: new Date(2019, 6, 5), end: new Date(2019, 6, 10)}, formatOptions: {dateStyle: 'medium'}},
+  name: 'RangeValue<Date>'
+};
+
+export let TimeRange: StaticFieldStory = {
+  ...Default,
+  args: {...Default.args, value: {start: new Time(9, 45), end: new Time(10, 50)}, formatOptions: {timeStyle: 'short'}},
+  name: 'RangeValue<Time>'
+};
+
+export let Number: StaticFieldStory = {
+  ...Default,
+  args: {...Default.args, value: 10},
+  name: 'Number'
+};
+
+export let NumberRange: StaticFieldStory = {
+  ...Default,
+  args: {...Default.args, value: {start: 10, end: 20}},
+  name: 'RangeValue<Number>'
+};
```

Comment:
For some reason this StaticField takes more horizontal room than it needs, something to note for a followup:
<img width="437" alt="image" src="https://user-images.githubusercontent.com/9661127/185266442-0cf8f4f8-c557-4f76-b6e6-29b9a450351a.png">

Reply:
It doesn't for me?

Reply:
this seems to happen when `labelPosition = 'top'` and `labelAlign = 'end'`, but it doesn't seem to be a problem when labelPosition is not specified. it looks like `labelPosition = top` adds `spectrum-Field-positionTop` class on the outermost div which sets the width to some default size.